Bottom: 26fcaa26995fe8b04408d1568f23d3c4a0dcad71
Top:    f7735041583683b606c41fcda88d553dc22b41be
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-01 19:30:09 +0000

Refresh of split-drc-routines-out-of-find

---

diff --git a/src/find.c b/src/find.c
index 5471f11..c38c967 100644
--- a/src/find.c
+++ b/src/find.c
@@ -65,30 +65,18 @@
 #include "config.h"
 #endif
 
-#include <stdlib.h>
-#ifdef HAVE_STRING_H
-#include <string.h>
-#endif
-#include <math.h>
 #include <setjmp.h>
 #include <assert.h>
 
-#ifdef HAVE_SYS_TIMES_H
-#include <sys/times.h>
-#endif
-
 #include "global.h"
 
-#include "crosshair.h"
 #include "data.h"
 #include "draw.h"
 #include "error.h"
 #include "find.h"
-#include "mymem.h"
 #include "misc.h"
 #include "rtree.h"
 #include "polygon.h"
-#include "pcb-printf.h"
 #include "search.h"
 #include "set.h"
 #include "undo.h"
